### DISCUSSION TITLE
How to add header when sending request.get

### DISCUSSION THREAD

**Discussion by uteminhduc** [FlareSolverr/FlareSolverr#977] [General]
Hi,
I read the docs but can't see any method to send a header attached to a request as cookies,

my `request.get` sample like that
```json
{
  "cmd": "request.get",
  "url": "https://api.sampleapis.com/beers/ale",
  "headers": {
    "Authorization": "JWT blabla"
  }
}
```
Please support that feature.

**Comment by Anonymous-AAA** [comment]
I hope they implement this feature.This project works for the time being 
https://github.com/Zelak312/flaresolverr-mitm-proxy

**Comment by ilike2burnthing** [comment]
#266
